src/pages/ClusterPage/explore.js

```diff
@@ -129,7 +129,7 @@ export default class ExploreCluster extends Component {
 
 			mainURL = `${mainURL}/${nestedRoute}`;
 		}
-		const url = `${mainURL}/?url=${arcURL}&username=${this.state.arc.username}&password=${this.state.arc.password}&cluster=${this.state.cluster}&header=false&showHelpChat=false`;
+		const url = `${mainURL}/?url=${arcURL}&username=${this.state.arc.username}&password=${this.state.arc.password}&cluster=${this.state.cluster}&showHelpChat=false&showProfile=false`;
```

Comment:
how to test? checkout `feat/search-settings` on `arc-dashboard` and replace above url with:

```
const url = `http://localhost:3333/?url=http://localhost:8000&username=foo&password=bar&cluster=${this.state.cluster}&showHelpChat=false&showProfile=false`;
```